java/java/JavaParser.g4

```diff
@@ -44,6 +44,7 @@ packageDeclaration
 
 importDeclaration
     : IMPORT STATIC? qualifiedName ('.' '*')? ';'
+    | ';'
```

Comment:
Please simplify in the following way:

```suggestion
    : (IMPORT STATIC? qualifiedName ('.' '*')?)? ';'
```

Reply:
I don't think this makes the rule easier to read/process.